### ISSUE TITLE
HX711 Integration

### ISSUE THREAD

**Issue by jung-youjin** [YeonsangYoon/embedded-system-project#2]
Hello there! I was wondering if you managed to integrate HX711 into Jetson board successfully! Thanks!:)

**Comment by YeonsangYoon** [comment]
No, we failed to integrate the HX711 into the Jetson board. There was a problem with Jetson Board's GPIO, so the HX711 was used separately through Raspberry Pi.

**Comment by jung-youjin** [comment]
@YeonsangYoon 키보드에 한국어가 안돼서 영어로 문의 드린 점 죄송합니다 허허.... 그렇게 하셨군요!!! 감사합니다🙏🏼 혹시 젯슨 위에서 어디까지 테스트해보셨나요...? 저 같은 경우는 계속 0이 떴는데, 어디까지 해보셨는지 궁금해서요!

**Comment by YeonsangYoon** [comment]
계속 GPIO 바꿔가면서 테스트를 해봤는데 인식이 안되더라고요. 저도 계속 값이 0이 떴습니다... 결국 라즈베리파이로 값을 읽어서 젯슨으로 보냈습니다
python jetson gpio 모듈을 좀 살펴봐도 크게 문제가 없어보였습니다.

**Comment by jung-youjin** [comment]
@YeonsangYoon 그렇군요..!!! 도움 주셔서 정말 감사합니다🙏🏼
